app/models/user.rb

```diff
@@ -20,4 +22,16 @@ class User
   def initialize(options = {}); end
 
   def save!; end
+
+  def attributes
```

Comment:
is this devise method? I believe we could then just do
```
def attributes
    super.merge({roles: nil, permissions: nil})
end
```
if that's all we're doing here